tests/examples/pacer/nef/s3/mdb_4.json

```diff
@@ -0,0 +1,90 @@
+{
+  "appellate": false,
+  "contains_attachments": true,
+  "court_id": "mdb",
+  "dockets": [
+    {
+      "case_name": "US Trustee - Greenbelt, 11 v. Martin",
+      "date_filed": null,
+      "docket_entries": [
+        {
+          "date_filed": "2024-04-12",
+          "description": "Adversary case 24-00085. (41 (Objection / revocation of discharge - 727(c),(d),(e))) Complaint by Lisa Yonka Stevens on behalf of US Trustee - Greenbelt 11 against Charles Martin. The filing fee is EXEMPT - Filed on behalf of the United States. (Stevens, Lisa)",
+          "document_number": "1",
+          "document_url": "https://ecf.mdb.uscourts.gov/doc1/092051683160?pdf_header=&magic_num=64349679&de_seq_num=3&caseid=783305",
+          "pacer_case_id": "783305",
+          "pacer_doc_id": "092051683160",
+          "pacer_magic_num": "64349679",
+          "pacer_seq_no": "3",
+          "short_description": "Complaint - AP - US Trustee"
```

Comment:
I would guess that this should just be `complaint`. Check the document history report?